config.toml

```diff
@@ -13,3 +13,34 @@ allowed-github-orgs = [
     "rust-dev-tools",
     "rust-embedded",
 ]
+
+permissions-bors-repos = [
+    "cargo",
+    "chalk",
+    "clippy",
+    "compiler-builtins",
+    "crater",
+    "crates-io",
+    "hashbrown",
+    "miri",
+    "libc",
+    "regex",
+    "rls",
+    "rust",
+    "rustlings",
+    "rustup-rs",
+    "semverver",
+    "stdarch",
+    "team",
+    "vscode-rust",
+]
+
+permissions-bools = [
```

Comment:
```suggestion
permissions-tools = [
```

Reply:
I kinda prefer to keep it as `bools`, since it will also eventually include permissions that are not tools.